src/rpc/mining.cpp

```diff
@@ -607,7 +607,6 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
     }
 
     UniValue aux(UniValue::VOBJ);
-    aux.pushKV("flags", HexStr(COINBASE_FLAGS.begin(), COINBASE_FLAGS.end()));
```

Comment:
Not sure what the implications of removing this field are, as I don't have strong knowledge of the `getblocktemplate` BIP. Though, when this is removed, the documentation in `RPCHelpMan` should be updated as well. And maybe `aux` can be removed as well? 
cc @luke-jr

Reply:
It looks like "flags" is entirely missing in the BIPs.

Reply:
(Also not supported by either Eloipool nor libblkmaker)

Reply:
ok, but `coinbaseaux` is: https://en.bitcoin.it/wiki/BIP_0022#Specification

So I guess `aux` should stay but be kept empty.